modules/ROOT/pages/to-change-raml-version.adoc

```diff
@@ -20,6 +20,8 @@ you add a new version to an existing asset.
 
 For an HTTP API, because the asset is only an endpoint, it makes no sense to increase the patch or minor versions in every new publication. HTTP APIs therefore only have major version increases and also changes to the API version.
 
+NOTE: New asset versions count toward your organization's xref:to-create-an-asset.adoc#asset-limits[asset limit].
+
 == Deploy a New API Version
 
 You can deploy an API asset directly into Exchange using the *New asset* button. In the *Publish a new asset* window, the Advanced section lets you specify the version of the asset 
```

Comment:
The title specifies a new version, but the first sentence only an API asset. Which is the most precise? Standardize to one wording.

Reply:
lets you specify > enables you to specify

Reply:
Note that compacted text should be edited at some point as well. I'm guessing that you don't have time for this right now, and I know that I do not! But maybe set yourself a JIRA.

Reply:
Standardized to "asset" and corrected to "enables you to specify." Yes, we'll edit the full Exchange docs during the Exchange reorganization.